common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -125,6 +126,11 @@ public boolean isVeteran() {
         return isVeteran;
     }
 
+    public boolean isHunted() {
+        Matcher m = LoreUtils.matchLoreLine(McUtils.inventory().getItem(SOUL_POINT_SLOT), 8, HUNTED_PATTERN);
```

Comment:
Can't you just use the diamond axe check here? I'm not sure but I'm guessing this was what @JamieCallan117 meant... It will make the method much simpler.